### PR TITLE
create suggest question of current_user

### PR DESCRIPTION
**Pull request by vuducluan**

### CODE REVIEW

File: app/views/questions/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<%= form_for @question, class: "form-group" do |f| %>
+  <%= f.label :subject_id %>
+  <%= f.collection_select :subject_id, Subject.all, :id, :name,
```
Comment (hellovietnam93) [inline app/views/questions/_form.html.erb]: register a variable in controller

File: config/routes.rb [hunk 2]
```diff
@@ -9,6 +9,6 @@
     resources :questions
   end
   resources :users, only: [:index, :update]
-  resources :questions, only:[:index, :show]
+  resources :questions, only:[:index, :show, :create, :new]
```
Comment (hellovietnam93) [inline config/routes.rb]: except

**Comment by hellovietnam93** [comment]
OK

File: app/views/questions/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<%= nested_form_for @question, class: "form-group" do |f| %>
+  <%= f.label :subject_id %>
+  <%= f.collection_select :subject_id, Subject.all, :id, :name,
```
Comment (dieunb) [inline app/views/questions/_form.html.erb]: đặt biến trong controller em nhé `Subject.all`

**Pull request merged**